### PR TITLE
Minimal seminar announcement

### PR DESCRIPTION
**Pull request by xabomon**
This is a minimal page announcing the VSF seminar series.

- [x] Agree on the seminar title
- [x] Add description of event
- [x] Add goal of event
- [x] Improve table of confirmed speakers with time of event
- [x] Add registration procedure

### CODE REVIEW

**Comment by xabomon** [comment]
Add registration form before final merge/

**Comment by akhmerov** [comment]
Yep, that's the idea.

File: long_range_colloquium.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Long Range Colloquium
+
+We aim to provide a platform for talks about latest developments in the fields of condensed matter and quantum information.
```
Comment (giulia-pacchioni) [inline long_range_colloquium.md:3]: I suggest being more assertive in this first sentence: we present a series of seminars about the latest developments in...
At this point we don't aim to provide a platform, we are using it :)
I'm new to GitHub, so let me know if I should have done something different from leaving a comment.

Comment (akhmerov) [reply long_range_colloquium.md]: All fixed, and this is exactly the right way to go.

**Comment by akhmerov** [comment]
I'm looking at the registration procedure.

Here's the eventbrite event: https://www.eventbrite.com/e/vsf-long-range-colloquium-tickets-101807751726

Here's the google form: https://forms.gle/W1SBnMbD8ypbXrPZ7

I'm leaning towards the google form: the amount of clicks and fields to fill in is about 2-3x lower.

Any opinions? @babaks77 @xabomon @giulia-pacchioni @greplova @vfatemi?

**Comment by akhmerov** [comment]
I pushed the registration link to google form for now, but happy to change to eventbrite if that's preferred.

Other than that last bit, all is ready to merge I think.

**Comment by vfatemi** [comment]
I agree with using google form for several already stated reasons. 

I edited the google form: 

- cleaned up the announcement text
- added a name entry (mandatory)
- checkbox for acknowledging the instructions now mandatory

**Comment by babaks77** [comment]
I also like the google form. Thanks!

**Pull request merged**